Kernel/Process.cpp

```diff
@@ -712,6 +712,18 @@ int Process::sys$gethostname(char* buffer, ssize_t size)
     return 0;
 }
 
+int Process::sys$sethostname(const char* hostname, ssize_t length)
+{
+    REQUIRE_PROMISE(stdio);
+    if (length < 0)
+        return -EINVAL;
```

Comment:
So what, any random process on the system can just go ahead and change hostname?

Reply:
My bad.